products/compute/api.yaml

```diff
@@ -130,6 +131,17 @@ objects:
           This should only be set when using an Internal address.
         values:
           - :GCE_ENDPOINT
+      - !ruby/object:Api::Type::Enum
+        name: purpose
+        exact_version: beta
+        description: |
+          The purpose of this resource, which can be one of the following values:
+          - GCE_ENDPOINT for addresses that are used by VM instances, alias IP ranges, internal load balancers, and similar resources.
+          This should only be set when using an Internal address.
```

Comment:
Should this field only be set when using an internal address, or should this field only be set to GCE_ENDPOINT when using an internal address? It's hard to tell which it is based on where the sentence is.